src/fetch.js

```diff
@@ -79,14 +83,17 @@ function parseResponse (fetchRes, body) {
     })
   }
   // check for rpc error
-  if (body.error) throw ethErrors.rpc.internal({
-    data: body.error,
-  })
+  if (body.error) {
+    throw ethErrors.rpc.internal({
+      data: body.error,
+    })
+  }
   // return successful result
   return body.result
 }
 
-function createFetchConfigFromReq({ req, rpcUrl, originHttpHeaderKey }) {
+function createFetchConfigFromReq ({ req, rpcUrl, originHttpHeaderKey }) {
+  // eslint-disable-next-line node/no-deprecated-api
   const parsedUrl = url.parse(rpcUrl)
```

Comment:
We should fix this in a follow-up: https://github.com/MetaMask/eth-json-rpc-middleware/issues/64